docs/books/ownCloud_Android_App_Manual.adoc

```diff
@@ -8,5 +8,4 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 1
 :icons: font
 :icon-set: octicon
-
-include::modules/ROOT/pages/index.adoc[leveloffset=+1]
+:module_base_path: modules/ROOT/pages/
```

Comment:
That should not be required?

Reply:
Is that a question or a statement?

Reply:
The modules path is defined on the cli, so this should be dropped.

Reply:
The cli module path definition is for Antora. This line is used by asciidoctor-pdf. However, it can be removed and set, instead, with an attribute in bin/cli. I'll make that change in the next commit.

Reply:
https://github.com/owncloud/android/pull/2601/files#diff-d687c1392f67d7207b33141ff4979ef5R150-R153 these paths ARE applied to asciidoctor-pdf. There is nothing about Antora at all.